### DISCUSSION TITLE
Disable trust-required filters on banking sites

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#3089] [Ideas]
AdGuard has something similar, but regarding the Certificate (in their desktop apps). Yes, I know adding banking sites to a list is tedious, but is there a better solution for security?

Or perhaps we could mirror / import the list from AdGuard, if there's a permissive license.

**Comment by Yuki2718** [comment]
Disabling https filtering on banking sites and disabling specific filter catogories is totally different thing, I believe we don't have trusted filters for banking site in the first place. Note user can still enable https filtering on them as the allowlist is editable. I'd also note deleting account after posting something can be regarded as very irresponsible behavior.

**Comment by gorhill** [comment]
One can use `banking.site#@#+js()` to disable all scriptlet injection on `banking.site`.
